lib/modules/dosomething/dosomething_api/resources/member_resource.inc

```diff
@@ -236,6 +275,38 @@ function _member_resource_create($request) {
 }
 
 /**
+ * Callback for User create.
+ *
+ * @param $request Array passed to the endpoint.
+ *   Possible keys:
+ *   - addr_street1
+ *   - addr_street2
+ *   - addr_city
+ *   - addr_state
+ * @return mixed Object of the newly created user if successful. String if errors.
+ */
+function _member_resource_update($id, $request) {
```

Comment:
just confirmed again w/ Adam & Aki all signups that have the address filled out should get Thumb Socks. No shipment entities needed!